src/M6Web/Bundle/ElasticsearchBundle/DataCollector/ElasticsearchDataCollector.php

```diff
@@ -86,4 +86,23 @@ public function reset()
         ];
     }
 
+    /**
+     * Converts a PHP variable to a string or
+     * Converts the variable into a serializable Data instance.
+     *
+     * The convert action depend on method available in the sf DataCollector class.
+     * In sf >= 4, the DataCollector::varToString() doesn't exists anymore
+     *
+     * @param mixed $var
+     *
+     * @return mixed
+     */
+    protected function stringifyVariable($var)
+    {
+        if (method_exists($this, 'varToString')) {
+            return $this->varToString($var);
+        } else {
+            return $this->cloneVar($var);
```

Comment:
I'm not sure that this will really return a `string` as your method indicates, but it cannot do any harm anyway 👍